frameworks/hdfs/alerts/HDFS_alert_rules.yml

```diff
@@ -0,0 +1,41 @@
+groups:
+- name: critical
+  rules:
+
+  - alert: InstanceDown
+    expr: up == 0
+    for: 5m
+    labels:
+      severity: page
+    annotations:
+      summary: "Instance {{ $labels.instance }} down"
+      description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes."
+
+  # Alert for any instance that has a median request latency > 1s.
+  - alert: APIHighRequestLatency
+    expr: api_http_request_latencies_second{quantile="0.5"} > 1
+    for: 1s
+    annotations:
+      summary: "High request latency on {{ $labels.instance }}"
+      description: "{{ $labels.instance }} has a median request latency above 1s (current value: {{ $value }}s)"
```

Comment:
Beyond repeating what the alert is signaling is there some kind of recommendation we can make to operators to check when this alert fires?